kernel/kernel-api/src/main/java/io/delta/kernel/internal/util/FileNames.java

```diff
@@ -25,6 +25,11 @@ public final class FileNames {
 
   private FileNames() {}
 
+  public enum DeltaLogFileType {
```

Comment:
I believe there is a similar enum in log replay somewhere. See if we can consolidate them.

Reply:
DeltaLogFile.java class has an enum for each specific type of delta/checkpoint file ... which is a bit more specific than what this enum is using it for. I'd encourage us to move forwards with this PR and look into this later.